### ISSUE TITLE
Fixed navbar size changes with page zoom in some browsers, making perfect body padding impossible.

### ISSUE THREAD

**Issue by kybishop** [twbs/bootstrap#18495]
This is in Bootstrap 4. Changing the font size when using a fixed navbar results in either a gap or overlap between navbar and content, with no clear way to perfectly pad body content. I have confirmed this issue with both Chrome and Safari.

This appears to be the result of the navbar changing some, but not all, sizes when font size increases or decreases. The padding top/bottom is a static 8 px, but the height is variable, meaning setting the padding-top on the body to rems or px doesn't quite work.

Example (make sure to change your font size to see issues!): http://jsfiddle.net/mz85wuck/

**Comment by twbs-lmvtfy** [comment]
Hi @kybishop!

You appear to have posted a live example (http://fiddle.jshell.net/mz85wuck/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 41, column 3: [W012](https://github.com/twbs/bootlint/wiki/W012): `.navbar`'s first child element should always be either `.container` or `.container-fluid`

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by kybishop** [comment]
Unless the v4-alpha docs are missing some content, the above linting error only applies to Bootstrap 3.

**Comment by cvrebert** [comment]
> make sure to change your font size to see issues!

Are you referring to using the browser's Zoom In/Out feature?

**Comment by kybishop** [comment]
@cvrebert Exactly. Command+plus and command+minus on Mac OS X

**Comment by cvrebert** [comment]
We tend to not support zooming: http://v4-alpha.getbootstrap.com/getting-started/browsers-devices/#browser-zooming

**Comment by cvrebert** [comment]
> Unless the v4-alpha docs are missing some content, the above linting error only applies to Bootstrap 3.

Indeed; Bootlint's version detection isn't perfect unfortunately.

**Comment by cvrebert** [comment]
Updated to v4.0.0-alpha.2: http://jsfiddle.net/cvrebert/v2q0nfL2/

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://fiddle.jshell.net/cvrebert/v2q0nfL2/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 41, column 3: [W012](https://github.com/twbs/bootlint/wiki/W012): `.navbar`'s first child element should always be either `.container` or `.container-fluid`

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by kybishop** [comment]
> We tend to not support zooming: http://v4-alpha.getbootstrap.com/getting-started/browsers-devices/#browser-zooming

Ah I missed that. You may close this issue if you think it is not worth fixing/requires too many hacky workarounds.

**Comment by cvrebert** [comment]
It's up to @mdo to make that determination.

**Comment by cvrebert** [comment]
@mdo What's the verdict here? WontFix?

**Comment by mdo** [comment]
Closing as a won't fix.
